bmrg-flow/templates/deployment-service.yaml

```diff
@@ -120,10 +120,10 @@ spec:
       - name: apm-newrelic
         emptyDir: {}
       {{- end }}
-      {{- if and (eq $k "controller") (eq $values.workers.logging.type "elastic" ) }}
+      {{- if or (and (eq $k "controller") (eq $values.workers.logging.type "elastic" )) ( $.Values.database.mongodb.tls.enable ) }}
       - name: secret-volume
         secret:
-          secretName: {{ $values.workers.logging.secret }}
+          secretName: {{ include "bmrg.name" (dict "context" $ "tier" "certs" ) }}
```

Comment:
@gchickma this relates to the question on the secret file. We shouldn't be copying and storing the contents, it should just reference an existing secret created by the installation of Elastic or Mongo